### PR TITLE
add recipe for liip/theme-bundle:1.5.*

### PR DESCRIPTION
**Pull request by lsmith77**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by lsmith77** [comment]
now can I ensure that `templating` is also enabled?

File: liip/theme-bundle/1.5/config/packages/theme.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+liip_theme:
+    themes: ['default']
+    active_theme: 'default'
+    load_controllers: false
```
Comment (lsmith77) [inline liip/theme-bundle/1.5/config/packages/theme.yaml]: ok .. that doesn’t seem to do the trick :-/

File: liip/theme-bundle/1.5/config/packages/framework.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+    templating:
+        engines: ['twig']
```
Comment (lsmith77) [inline liip/theme-bundle/1.5/config/packages/framework.yaml]: ok .. that doesn’t seem to do the trick :-/

Comment (lsmith77) [reply liip/theme-bundle/1.5/config/packages/framework.yaml]: would it work if I fold it into the theme.yaml?

File: liip/theme-bundle/1.5/post-install.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+  <bg=blue;fg=white>         Getting started using </><bg=blue;fg=white;options=bold>liip/theme-bundle</><bg=blue;fg=white>          </>
+
+  <options=bold>Configure</> the bundle and routes:
+    1. You <options=bold>MUST</> verify the configuration in <comment>%CONFIG_DIR%/packages/theme.yaml</>.
+    2. You <options=bold>MAY</> configure additional features, see the README.md for instructions.
+    3. You <options=bold>MAY</> configure the routes as specified in the README.md if you enable the controller.
```
Comment (Taluu) [inline liip/theme-bundle/1.5/post-install.txt]: You should remove the post-install

File: liip/theme-bundle/1.5/config/packages/framework.yaml (hunk 2 above)
Comment (fabpot) [reply liip/theme-bundle/1.5/config/packages/framework.yaml]: Using the Symfony Templating layer as done here is "forbidden" in Symfony 3.4/4.0. Supporting Twig directly is what is epxected in recipes.

Comment (lsmith77) [reply liip/theme-bundle/1.5/config/packages/framework.yaml]: ok .. which requires a major refactoring of this Bundle .. which I am currently not planning to work on (see https://github.com/liip/LiipThemeBundle/issues/181).

so I guess I will close this PR

**Pull request closed without merging**